studio/src/windowManagement/contentWindows/ContentWindowBuildView/ContentWindowBuildView.js

```diff
@@ -82,15 +82,15 @@ export class ContentWindowBuildView extends ContentWindow {
 			hasDownArrow: true,
 			colorizerFilterManager,
 			onClick: () => {
-				const studio = getStudioInstance();
-				const projectSettings = studio.projectManager.projectSettings;
-				const assetManager = studio.projectManager.assetManager;
-				if (!projectSettings || !assetManager) return;
-				const popover = studio.popoverManager.createPopover();
-				// eslint-disable-next-line no-new
-				new EntryPointManager(popover, projectSettings, assetManager, this.persistentData);
-
-				popover.setNeedsCurtain(false);
```

Comment:
The curtain was disabled to allow you to drag assets from the project window to the entry point dropdown gui.
Without this the popover will close when you try to drag assets.
Ideally a test should have caught this, so I've opened up #504